model/import/Repository/CsvTemplateRepository.php

```diff
@@ -35,13 +35,14 @@ public function findById(string $id): ?TemplateInterface
         if ($id === self::DEFAULT) {
             return new CsvTemplate(
                 TemplateRepositoryInterface::DEFAULT,
-                TemplateRepositoryInterface::DEFAULT_DEFINITION
+                TemplateRepositoryInterface::DEFAULT_DEFINITION,
+                TemplateRepositoryInterface::DEFAULT_XML
```

Comment:
I repository plays a different role. Here we search for CSV templates definitions. 

For parsing templates into items should be another class. This class should not know about output parsing